### DISCUSSION TITLE
devices were not found

### DISCUSSION THREAD

**Discussion by alinechiang** [marcus-crane/october#136] [Q&A]
i downloaded the latest version from homebrew and tried the universal version too.
but calibre is able to see it
<img width="789" alt="image" src="https://github.com/user-attachments/assets/1b0449cd-a3f2-4964-bc6e-7a91d25dbd92" />

**Comment by marcus-crane** [comment]
Hey Aline,

As a start, what model of Kobo do you have? It shouldn't make much of a difference as even newer models (that October doesn't "know" about) should be supported but there's a chance new models may slightly change some details that cause them to not be detected properly.

**Comment by alinechiang** [comment]
hi it’s a kobo libra color

Aline
***@***.*** ***@***.***>*
Shanghai/Canada

**Reply by marcus-crane** [reply]
Ah ok, that shouldn't be an issue then I think since it's a known device for October. From the screenshot, it seems like you're on macOS? If you open up Finder and check the volumes section in the sidebar, is the device named "KOBOeReader"?

In rare cases, it can be named stuff like "KOBOeReader (2)". I would be surprised if that's happening but I'm not sure if Calibre supports that case so it's worth checking.

**Comment by alinechiang** [comment]
it was there, but i renamed it.
when i first downloaded it it worked with the original name
do you think that caused the issie?
and also i used the sqlite file instead and worked lol

On Sat, 14 Dec 2024 at 18:31, Marcus Crane ***@***.***> wrote:

> Ah ok, that shouldn't be an issue then I think since it's a known device
> for October. From the screenshot, it seems like you're on macOS? If you
> open up Finder and check the volumes section in the sidebar, is the device
> named "KOBOeReader"?
>
> In rare cases, it can be named stuff like "KOBOeReader (2)". I would be
> surprised if that's happening but I'm not sure if Calibre supports that
> case so it's worth checking.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/marcus-crane/october/discussions/136#discussioncomment-11569294>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALETPI2XEVID64MGY3BXMOL2FS5URAVCNFSM6AAAAABTTNP2PKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNJWHEZDSNA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


Aline
***@***.*** ***@***.***>*
Shanghai/Canada
